pkgs/development/python-modules/eth-abi/default.nix

```diff
@@ -8,56 +8,47 @@
   hypothesis,
   parsimonious,
   pytestCheckHook,
-  pythonOlder,
+  pytest-xdist,
+  setuptools,
 }:
 
 buildPythonPackage rec {
   pname = "eth-abi";
-  version = "4.1.0";
-  format = "setuptools";
-  disabled = pythonOlder "3.7";
+  version = "5.1.0";
```

Comment:
Please set `pyproject = true` in every python package.

Reply:
done